CHANGES.md

```diff
@@ -29,6 +29,10 @@ OpenSSL 3.4
 
 ### Changes between 3.3 and 3.4 [xx XXX xxxx]
 
+ * Add feature to retrieve configured TLS signature algorithms.
```

Comment:
I'd mention something about the list app

Reply:
So added in https://github.com/openssl/openssl/pull/24821/commits/bff436b74df85a6acdf34e31415a31f4fc145f21.